fe/fe-core/src/main/java/com/starrocks/service/InformationSchemaDataSource.java

```diff
@@ -425,6 +427,10 @@ public static TGetTablesInfoResponse generateTablesInfoResponse(TGetTablesInfoRe
 
         MetadataMgr metadataMgr = GlobalStateMgr.getCurrentState().getMetadataMgr();
 
+        ConnectContext connectContext = new ConnectContext();
+        connectContext.setQueryId(UUIDUtil.genUUID());
+        connectContext.setThreadLocalInfo();
```

Comment:
Better to remove the connectContext from threadlocal in the end to avoid thread pollution.